app/renderer/components/tabs/content/tabIcon.js

```diff
@@ -30,12 +30,22 @@ class TabIcon extends ImmutableComponent {
       }
     })
 
+    let altProps
+    if (!this.props.symbol) {
+      altProps = {
+        'data-test-id': this.props['data-test-id'],
+        'data-test2-id': this.props['data-test2-id']
+      }
+    }
+
     return <div
       className={this.props.className}
       data-test-favicon={this.props['data-test-favicon']}
       onDragStart={this.props.onDragStart}
       draggable={this.props.draggable}
-      onClick={this.props.onClick}>
+      onClick={this.props.onClick}
+      {...altProps}
```

Comment:
this is needed, because we add `data-test-id` to span element, but click is added to div element and if you want to do `click` in the test you need to have `data-test-id` on div as well.